service/src/main/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataService.java

```diff
@@ -68,7 +67,7 @@ public RoboticsData prepareRoboticsData(final CaseData caseData, final Long case
                 && isNotBlank(caseData.getAllocationProposal().getProposal())
                 ? caseData.getAllocationProposal().getProposal() :  null)
             .issueDate(isNotEmpty(caseData.getDateSubmitted())
-                ? dateFormatterService.formatLocalDateToString(caseData.getDateSubmitted(), "dd-MM-yyyy") : "")
+                ? formatLocalDateToString(caseData.getDateSubmitted(), "dd-MM-yyyy") : "")
```

Comment:
constant? Although this might actually be a new constant, don't think we have one for that format yet.